<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorADS1219Dialog</class>
 <widget class="QDialog" name="SensorADS1219Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="i2cLabel">
     <property name="text">
      <string>I2C address</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="ch1Box">
     <property name="text">
      <string>Channel 1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="ch1CalBox">
     <property name="text">
      <string>Channel 1 calibration</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="ch2Box">
     <property name="text">
      <string>Channel 2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="ch2CalBox">
     <property name="text">
      <string>Channel 2 calibration</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="ch3Box">
     <property name="text">
      <string>Channel 3</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="ch3CalBox">
     <property name="text">
      <string>Channel 3 calibration</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="ch4Box">
     <property name="text">
      <string>Channel 4</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="ch4CalBox">
     <property name="text">
      <string>Channel 4 calibration</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="gainLabel">
     <property name="text">
      <string>Gain setting</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="gainBox">
     <item>
      <property name="text">
       <string>Gain 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gain 4</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="vrefLabel">
     <property name="text">
      <string>Reference voltage</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="vrefBox">
     <item>
      <property name="text">
       <string>Internal reference</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>External reference</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="gperiodBox">
     <property name="text">
      <string>Use global measurement policy</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="gperiodButton">
     <property name="text">
      <string>Edit global measurement policy</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="confirmBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDoubleSpinBox" name="periodBox">
     <property name="suffix">
      <string notr="true">s</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>6553.600000000000364</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="addressBox">
     <property name="prefix">
      <string>0x</string>
     </property>
     <property name="minimum">
      <number>64</number>
     </property>
     <property name="maximum">
      <number>79</number>
     </property>
     <property name="displayIntegerBase">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="periodLabel">
     <property name="text">
      <string>Measurement period</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>confirmBox</sender>
   <signal>accepted()</signal>
   <receiver>SensorADS1219Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirmBox</sender>
   <signal>rejected()</signal>
   <receiver>SensorADS1219Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
